neurovault/apps/statmaps/tests/test_celery.py

```diff
@@ -0,0 +1,31 @@
+# in one terminal window: 
```

Comment:
how difficult would it be to turn this into a real django test that does not need to be run automatically? https://docs.djangoproject.com/en/1.7/topics/testing/ http://docs.celeryproject.org/en/2.5/django/unit-testing.html